transiXNOR/plotting.py

```diff
@@ -125,42 +125,69 @@
 
 if (PLOT_CURRENT or PRINT_CURRENT_ONLY):
 	# Fig. 1
-	cur = np.abs(np.load(model_path + '/current_20.npy'))
+	cur = np.abs(np.load(model_path + '/current_10.npy'))
```

Comment:
Can you make VDS as an variable? So that if you have to plot at different VDS, you only need to change VDS value once.